docs/network-drivers.md

```diff
@@ -148,6 +150,12 @@ Register ports just like the powerconnect driver. e.g. ``gi1/0/5``.
 
 ### Nexus driver
 
+#### Switch preperatio
+
+A few commands are necessary to run on the switch before it can be used with HIL.
+
+1. This switch uses ssh for connection. Be sure that ssh is enabled on the switch.
```

Comment:
Given that we just have the one thing here, I might cut out the "A few commands..." bit from this section and the leading `1.`; it can just be one sentance.